Sources/S5TF/utils/S5TFUtils.swift

```diff
@@ -79,7 +79,7 @@ public struct S5TFUtils {
     ///     ```
     ///
     @discardableResult
-    static public func extract(fileAt: URL) throws -> (out: String?, status: Int32) {
```

Comment:
`@discardableResult` is superfluous when nothing is returned. Please return something though.